terraform/ci/main.tf

```diff
@@ -14,75 +14,38 @@
  * limitations under the License.
  */
 locals {
-  github_slug = "abcxyz/jvs"
-}
-
-resource "google_project_service" "serviceusage" {
-  project            = var.project_id
-  service            = "serviceusage.googleapis.com"
-  disable_on_destroy = false
+  github_owner_id = 93787867  # abcxyz
+  github_repo_id  = 479173136 # abcxyz/jvs
 }
 
 resource "google_project_service" "services" {
-  project = var.project_id
   for_each = toset([
-    "cloudresourcemanager.googleapis.com",
-    "iamcredentials.googleapis.com",
-    "artifactregistry.googleapis.com",
     "cloudkms.googleapis.com",
   ])
+
+  project            = var.project_id
   service            = each.value
   disable_on_destroy = false
-
-  depends_on = [
-    google_project_service.serviceusage,
-  ]
-}
-
-resource "google_artifact_registry_repository" "image_registry" {
-  provider = google-beta
-
-  location      = var.artifact_registry_location
-  project       = var.project_id
-  repository_id = "docker-images"
-  description   = "Container Registry for the images."
-  format        = "DOCKER"
-  depends_on = [
-    google_project_service.services["artifactregistry.googleapis.com"],
-  ]
-}
-
-resource "google_service_account" "gh-access-acc" {
-  project      = var.project_id
-  account_id   = "gh-access-sa"
-  display_name = "GitHub Access Account"
 }
 
 // IAM roles needed to run tests.
 resource "google_project_iam_member" "gh_access_acc_iam" {
   for_each = toset(var.ci_iam_roles)
   project  = var.project_id
   role     = each.key
-  member   = "serviceAccount:${google_service_account.gh-access-acc.email}"
+  member   = module.github_ci_infra.service_account_member
 }
 
-module "workload-identity-federation" {
-  source      = "github.com/abcxyz/pkg//terraform/modules/workload-identity-federation"
-  project_id  = var.project_id
-  github_slug = local.github_slug
+module "github_ci_infra" {
```

Comment:
Should this belong to infra pkg?

Reply:
discussed offline, resolving.

Reply:
Discussed offline. If this is a new project, I'd make it in the infra repo.